src/rust/e2e-tests/tests/sysmon_log_e2e_test.rs

```diff
@@ -0,0 +1,177 @@
+#![cfg(feature = "integration_tests")]
+mod test_utils;
+
+use bytes::Bytes;
+use kafka::{
+    config::ConsumerConfig,
+    test_utils::topic_scanner::KafkaTopicScanner,
+};
+use plugin_work_queue::test_utils::scan_for_plugin_message_in_pwq;
+use rust_proto::graplinc::grapl::{
+    api::{
+        event_source::v1beta1::CreateEventSourceRequest,
+        graph::v1beta1::GraphDescription,
+        pipeline_ingress::v1beta1::PublishRawLogRequest,
+        plugin_registry::v1beta1::{
+            DeployPluginRequest,
+            PluginMetadata,
+            PluginType,
+        },
+    },
+    pipeline::v1beta1::RawLog,
+};
+use test_context::test_context;
+use test_utils::context::E2eTestContext;
+use uuid::Uuid;
+
+use crate::test_utils::predicates::{
+    events_36lines_merged_graph_predicate,
+    events_36lines_node_identity_predicate,
+};
+
+mod test_fixtures {
+    use super::*;
+
+    /// Send 1 line (well, event) at a time
+    pub fn get_36_eventlog_xml_separate_lines() -> Result<Vec<String>, std::io::Error> {
+        let filename = "/test-fixtures/36_eventlog.xml"; // This path is created in rust/Dockerfile
```

Comment:
IMO we should just move that file under this package, or sysmon-generator + symlink to that from this package, and not rely on sync with the Dockerfile. That file is only used for e2e testing and doesn't need to be in a repo-level etc directory, right?

Reply:
i can very much see a world where perhaps another test that's not e2e would want to rely on these - in fact i've been in those scenarios before. I'd like to maybe punt on this until the future of the repo is a bit more obvious.